react/Checkbox/Checkbox.less

```diff
@@ -40,16 +40,16 @@
     .focus();
   }
 
-  .input:checked + .label & {
-    background-color: @sk-mid-gray-light;
+  .label & {
+    background-color: @sk-white;
   }
 }
 
 .checkMark {
   display: block;
   width: 100%;
   height: 100%;
-  fill: @sk-black;
+  fill: @sk-focus;
```

Comment:
Good point @michaeltaranto thought about that. We already had the design discussion re that and it was how I got it, so they always should be in sync...